### PR TITLE
[android] update readme for compiling with NDK 12b and SDK21

### PR DESCRIPTION
**Pull request by MartijnKaijser**
Fixes
http://trac.kodi.tv/ticket/16974

### CODE REVIEW

**Comment by phillipao** [comment]
Looks like 12b still requires ndk-dir but doesn't recognize --system. So for x86:

./make-standalone-toolchain.sh --ndk-dir=../.. --install-dir=<android-toolchain-x86>/android-21 --platform=android-21 --toolchain=x86-4.9 --arch=x86

File: docs/README.android [hunk 1]
```diff
@@ -133,25 +133,34 @@ common scenario.
...
+      --install-dir=<android-toolchain-arm>/android-21 --platform=android-21 \
+      --toolchain=arm-linux-androideabi-4.9
+
+Building for aarch64 architecture:
+
+    $ cd <android-ndk>
+    $ ls platforms
+    $ cd build/tools
+    $ ./make-standalone-toolchain.sh --ndk-dir=../../ \
+      --install-dir=<android-toolchain-arm>/android-21 --platform=android-21 \
```
Comment (fetzerch) [inline docs/README.android]: should the install-dir for aarch64 be `android-linux-aarch64-4.9`?

**Comment by fetzerch** [comment]
https://github.com/MartijnKaijser/xbmc/blob/9dd3894cfc649ed712edf234dd31e89544053d51/docs/README.android#L91 should be updated as well.

**Comment by MartijnKaijser** [comment]
updated

**Pull request merged**